lib/node_modules/@stdlib/math/base/special/fibonacci/src/main.c

```diff
@@ -0,0 +1,48 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/fibonacci.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/assert/is_integer.h"
+#include "stdlib/constants/float64/max_safe_nth_fibonacci.h"
+
+static const int64_t fibonacci_value[79] = {
```

Comment:
We can maybe go higher than `79` values given that we're returning a 64-bit signed integer. Should investigate what the maximum value can be.

Reply:
We decided to mirror the JS API. We can now disregard this comment.